projects/ngx-skeleton-loader/src/lib/ngx-skeleton-loader-config.types.ts

```diff
@@ -0,0 +1,24 @@
+import { InjectionToken } from '@angular/core';
+
+export interface NgxSkeletonLoaderConfig {
```

Comment:
```suggestion
import { InjectionToken } from '@angular/core';

export type NgxSkeletonLoaderConfigTheme = {
  // This is required since ngStyle is using `any` as well
  // More details in https://angular.io/api/common/NgStyle
  // tslint:disable-next-line: no-any
  [k: string]: any;
} | null;

export interface NgxSkeletonLoaderConfig {
  appearance?: 'circle' | 'line' | '';
  animation?: 'progress' | 'progress-dark' | 'pulse' | 'false' | false;
  theme?: NgxSkeletonLoaderConfigTheme;
  loadingText?: string;
  count?: number;
}

export const NGX_SKELETON_LOADER_CONFIG = new InjectionToken<NgxSkeletonLoaderConfig>('ngx-skeleton-loader.config');
```